src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -112,87 +112,99 @@ function ScrollDown(props) {
   );
 }
 
-function RetrieveBackgroundImage() {
+async function getDynamicAsset(url, callback, errorCB) {
```

Comment:
It's not a `callback` if your first arg isn't an `error` object, per node convention.  So I'd follow the pattern used in other Web APIs and do:

```js
async function getDynamicAsset(url, success, failure) {
   ...
}
```

Each of `success` and `failure` are functions that don't expect to receive an `error` object first.

Reply:
Haven't done true node in a while, I appreciate the convention tip. Corrected.